sdk/appconfiguration/azure-appconfiguration-provider/tests/test_async_provider.py

```diff
@@ -17,6 +18,7 @@ async def build_provider(
         return await load(connection_string=connection_string, trim_prefixes=trim_prefixes, selects=selects)
 
     # method: provider_creation
+    @pytest.mark.skip
```

Comment:
Why we skip it?

Reply:
As we know, all provider tests are failing. But this will block azure-appconfiguration samples to run across cloud in tests-weekly pipeline. Before waiting for their fix, I want to skip them now.

Reply:
This will skip the provider tests from running in public cloud too, right?

Reply:
Yes, skip running in all three clouds.